packages/tools/src/tools/segmentation/SphereScissorsTool.ts

```diff
@@ -71,6 +72,7 @@ class SphereScissorsTool extends BaseTool {
       configuration: {
         strategies: {
           FILL_INSIDE: fillInsideSphere,
+          ERASE_INSIDE_SPHERE: eraseInsideSphere,
```

Comment:
ERASES_INSIDE? to match the FILL_INSIDE?

Reply:
No, really bad idea as it means you can't use the strategy name independently.  Rather fix FILL_INSIDE to be FILL_INSIDE_SPHERE